ci/install-package.sh

```diff
@@ -222,17 +222,18 @@ then
     docker exec -t tue-env bash -c 'source ~/.bashrc; git -C "$TUE_SYSTEM_DIR"/src/"$PACKAGE" fetch origin "$REF_NAME"/"$PULL_REQUEST"/merge:PULLREQUEST'
 
     # Install the package completely
-    echo -e "\e[35m\e[1mtue-get install ros-$PACKAGE --test-depend --branch=PULLREQUEST\e[0m"
-    docker exec tue-env bash -c 'source ~/.bashrc; tue-get install ros-"$PACKAGE" --test-depend --branch=PULLREQUEST'
+    branch_string=${BRANCH:+" --try-branch=${BRANCH}"}
+    echo -e "\e[35m\e[1mtue-get install ros-$PACKAGE --test-depend${branch_string} --try-branch=PULLREQUEST\e[0m"
+    docker exec tue-env bash -c 'source ~/.bashrc; tue-get install ros-"${PACKAGE}" --test-depend --try-branch="${BRANCH}" --try-branch=PULLREQUEST'
```

Comment:
I think we should use the branch_string here as well

Reply:
We don't do this later in the script. Which you also commented. It doesn't matter when  the argument is empty. But is doesn't hurt to change it either

Reply:
Just for the sake for uniformity otherwise it only becomes a fancy print statement :)

Reply:
If we just drop in the `branch_string` variable? Will it still be resolved inside the docker?

Reply:
I hope so :laughing: